uint8 id_mcpf
uint8 id_mcpa
uint8 id_pip
uint8 id_dip
int16 vel_mcpf
int16 vel_mcpa
int16 vel_pip
int16 vel_dip